test/junit/scala/collection/mutable/BitSetTest.scala

```diff
@@ -153,6 +153,20 @@ class BitSetTest {
   @Test def buildFromRange(): Unit = {
     import scala.util.chaining._
     assert((1 to 1000).to(BitSet) == BitSet().tap(bs => (1 to 1000).foreach(bs.addOne)))
+  }
 
+  @Test def iteratorFrom(): Unit = {
+    val a = (0 to 127).to(BitSet)
```

Comment:
Almost all tests are based on `a`, the only exception being the one with `BitSet(0)`. `a` is quite regular with 2 full words. Can we have some tests with holes in the data or data that does not begin and end on a full word? Maybe use a ScalaCheck test instead of manually coming up with corner cases?

Reply:
Agreed, I've added ScalaCheck test.